app/controllers/subscribe.js

```diff
@@ -0,0 +1,99 @@
+import Controller from '@ember/controller';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+import { inject as service } from '@ember/service';
+import { RDS_TWITTER, APPS } from '../constants/urls';
+import { TOAST_OPTIONS } from '../constants/toast-options';
+
+export default class SubscribeController extends Controller {
+  @service login;
+  @tracked isFormOpen = false;
+  @tracked email = '';
+  @tracked phone = '';
+  @tracked userData = null;
+  @tracked isLoading = false;
+  @tracked showSubscriptionConfirmation = false;
+
+  RDS_TWITTER = RDS_TWITTER;
+
+  constructor() {
+    super(...arguments);
+    this.userData = this.login.userData;
+  }
+
+  get isLoggedIn() {
+    return this.login.isLoggedIn;
+  }
+
+  get isSubscribed() {
+    return this.login.userData.isSubscribed;
+  }
+
+  get isSubmitDisabled() {
+    return !this.email || (this.phone && !/^\+?\d*$/.test(this.phone));
```

Comment:
Currently we are checking here. I have discussed this with @ankushdharkar we will add country code library with flag here. I have created flag here.
https://github.com/Real-Dev-Squad/website-backend/issues/2210